isofit/test/test_pep8.py

```diff
@@ -1,46 +0,0 @@
-#! /usr/bin/env python3
```

Comment:
This test runs, but does not actually check any files. Manually running `$ pep8` against ISOFIT with the config file discovers 191 violations. The team seems to be OK with the current style, so I'm removing the test.

If the team has a desire to run `pep8`, I would recommend adding it to the linting CI job and `pre-commit` hook.